cb_number_of_holidays_report/report/holidays_count_report.py

```diff
@@ -0,0 +1,65 @@
+from odoo import api, fields, models, _
+from odoo.exceptions import UserError
+import math
+
+
+class HolidaysCountReport(models.AbstractModel):
+    _name = 'report.cb_number_of_holidays_report.report_holidays_count'
+
+    @api.model
+    def get_report_values(self, docids, data=None):
+        if not data.get('form'):
+            raise UserError(_(
+                "Form content is missing, this report cannot be printed."))
+
+        date_from = data['form']['date_from']
+        date_to = data['form']['date_to']
+
+        docs = []
+
+        for employee in self.env['hr.employee'].browse(
+                data['form']['employee_ids']):
+            holidays = self.env['hr.holidays'].search(
+                [('employee_id', '=', employee.id),
+                 ('type', '=', 'remove'),
+                 ('date_from', '<=', date_to),
+                 ('date_to', '>=', date_from),
+                 ('state', '=', 'validate'),
+                 ('count_in_hours', '=', False)
+                 ]
+            )
+
+            days_count = 0.0
+            date_from_day = fields.Datetime.from_string(date_from)
+            date_to_day = fields.Datetime.from_string(date_to)
+            for holiday in holidays:
+                if date_from >= holiday.date_from and (
```

Comment:
you should use https://github.com/odoo/odoo/blob/11.0/addons/hr_holidays/models/hr_holidays.py#L284

Reply:
Yes, I guess it's more clear, I can set employee to false and use this.